WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -882,11 +882,30 @@ - (UIAlertController *)makeAddSiteAlertController
                                                    handler:nil];
 
     [addSiteAlertController addAction:addSiteAction];
+
+    if ([Feature enabled: FeatureFlagEnhancedSiteCreation]) {
+        UIAlertAction *enhancedSiteAction = [UIAlertAction actionWithTitle:NSLocalizedString(@"Enhanced site creation", @"Enhanced site creation")
+                                                                style:UIAlertActionStyleDefault
+                                                              handler:^(UIAlertAction *action) {
+                                                                  [self enhancedSiteCreation];
+                                                              }];
+
+        [addSiteAlertController addAction:enhancedSiteAction];
+    }
+
     [addSiteAlertController addAction:cancel];
 
     return addSiteAlertController;
 }
 
+- (void)enhancedSiteCreation
```

Comment:
@nheagy raises an interesting point. 

Right now we have an `@objc` annotation in `SiteCreationWizardLauncher`. Creating a Swift-based extension would allow us to move the annotation, and it might make it clearer where we are exposing behavior to Objective-C.